dns/nameserver.py

```diff
@@ -247,6 +262,7 @@ def query(
             one_rr_per_rrset=one_rr_per_rrset,
             ignore_trailing=ignore_trailing,
             server_hostname=self.hostname,
+            verify=self.verify,
```

Comment:
Any reason this isn't also added to `async_query`?

Reply:
This is why code review is good :)